2019-10-20 00:45:05.766 [error] <0.543.0>@hackney_metrics:get_engine:26 gen_server tzdata_release_updater terminated with reason: bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26
2019-10-20 00:45:05.768 [error] <0.543.0>@hackney_metrics:get_engine:26 CRASH REPORT Process tzdata_release_updater with 0 neighbours crashed with reason: bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26
2019-10-20 00:45:05.768 [error] <0.224.0>@hackney_metrics:get_engine:26 Supervisor {<0.224.0>,'Elixir.Supervisor.Default'} had child 'Elixir.Tzdata.ReleaseUpdater' started with 'Elixir.Tzdata.ReleaseUpdater':start_link() at <0.543.0> exit with reason bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26 in context child_terminated
2019-10-20 22:07:09.188 [error] <0.1118.0>@hackney_metrics:get_engine:26 gen_server tzdata_release_updater terminated with reason: bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26
2019-10-20 22:07:09.190 [error] <0.1118.0>@hackney_metrics:get_engine:26 CRASH REPORT Process tzdata_release_updater with 0 neighbours crashed with reason: bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26
2019-10-20 22:07:09.191 [error] <0.224.0>@hackney_metrics:get_engine:26 Supervisor {<0.224.0>,'Elixir.Supervisor.Default'} had child 'Elixir.Tzdata.ReleaseUpdater' started with 'Elixir.Tzdata.ReleaseUpdater':start_link() at <0.1118.0> exit with reason bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26 in context child_terminated
